roboteam_ai/src/interface/drawer.cpp

```diff
@@ -31,6 +34,40 @@ std::vector<std::pair<Vector2, QColor>> Drawer::getGoToPosLuThPoints(int id) {
 
 }
 
+void Drawer::setKeeperPoints(int id,std::vector<std::pair<rtt::Vector2, QColor>> points) {
+    std::lock_guard<std::mutex> lock(mutex);
```

Comment:
give the mutex a proper name, like "keeperMutex" and "gtpMutex", use different names for different locks